easybuild/tools/package/utilities.py

```diff
@@ -134,7 +135,7 @@ def package_with_fpm(easyblock):
         '-s', 'dir',  # source
         '--version', pkgver,
         '--iteration', pkgrel,
-        '--description', quote_str(easyblock.cfg["description"]),
+        '--description', quote_str(eb_shell_quote(easyblock.cfg["description"])),
```

Comment:
You can see now that I've removed both quotes. If you pass the commands as a list it will handle doing the escaping properly.

It is necessary to pass each separate part of the command line as its own list entry though (so I had to also change the way I was handling the excludes and such).

I think I'm missing a test case...